docs/development/release.md

```diff
@@ -62,63 +60,54 @@ know how it ends up ;-)
 
 Release candidate versions must have the naming ``v2.9.0-rc1``, ``v2.9.0-rc2`` etc.
 
-### 1 - Create the release branch
+### Create the release
 
-Master branch being protected, we do our work on a dedicated branch that we call the release branch.
-Release branch naming must be ``<major>.<minor>``, so in our case ``2.9``:
+We have the script `misc/releaser.py` that provide an automated workflow to generate a new release/build.
 
-```shell
-git fetch
-git switch --create release/2.9 origin/master
-git push --set-upstream origin release/2.9
-```
-
-Note at this point the branch is even with master, hence we cannot create a Pull Request yet (see part 4).
+Note: The commits and release tag are annotated & signed, so you must have your GPG key
+at the ready (and this key should be configured in your github account).
 
-### 2 - Create release commits and tag
+To create a new release, simply execute:
 
 ```shell
-python misc/releaser.py build v2.9.0
+git fetch
+python misc/releaser.py build --base=origin/master v2.9.0
 ```
 
-The script will pause just before creating the release commit, this is so you
-can open [`HISTORY.rst`](/HISTORY.rst) and check the release note.
+> `git fetch` is used to update the remote ref, that is used to get the latest remote change on `master`.
+> We specify `--base=origin/master` because you are not enforce to be on the `master` branch to use that script.
```

Comment:
```suggestion
> `--base=origin/master` is used because you do not need to be on the `master` branch to use the script.
```

Reply:
I've applied the suggestion from @Ironicbay before you propose for that one, could re-review that part ?